src/features/common/Layout/Footer/index.tsx

```diff
@@ -31,28 +31,59 @@ export default function Footer() {
     setOpenModal(false);
   };
 
-  const FooterLinks = [
-    {
-      id: 1,
+  const [userLang,setUserLang] = React.useState('en');
+  React.useEffect(()=>{
+    if (typeof window !== 'undefined') {
+      let userLang = localStorage.getItem('language') || 'en';
```

Comment:
We already load the `language` from localStorage some lines below - can't we also add `setUserLang` down there at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/8eabaaa4a2fc5816a1de84ab391d34207fa6f0d6/src/features/common/Layout/Footer/index.tsx#L68-L71

Reply:
Now it has a difference as it is all lowercase :-) I renamed it from `userLang` to `footerLang` for some more clarity of the usage compared to `language`.